packages/stencil/src/components/datacom-radio/datacom-radio.tsx

```diff
@@ -55,14 +58,21 @@ export class DatacomRadio implements FormControl {
   private inputElement: HTMLInputElement;
 
   handleChange = () => {
+    console.log('handlechange', this.checked);
+
     this.checked = true;
   };
 
-  /**
-   * Check if control is valid (for form submit)
-   *
-   * @returns Promise of validity
-   */
+  // * control use for radio button group   */
```

Comment:
Please use multi line comment syntax for consistency with other function comment formats

```
/**
 * Comment goes here.
 */
```

Reply:
Fixed it.